### PR TITLE
Move some static methods from HiveMetadata to HiveUtil

### PR DESCRIPTION
**Pull request by MiguelWeezardo**

### CODE REVIEW

**Comment by raunaqmorarka** [comment]
Should we move these to HiveUtil instead ?

**Comment by MiguelWeezardo** [comment]
> Should we move these to HiveUtil instead ?

Great idea.

**Review by findepi — APPROVED** [review]

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java
Comment (phd3) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java:2867]: nit: I'd change this to static import

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHiveLocal.java
Comment (phd3) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHiveLocal.java:227]: nit: static import

**Review by phd3 — APPROVED** [review]
LGTM modulo a minor comment.

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java
Comment (findepi) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java:1056]: i think i was writing a comment about this but cannot see it now... sorry for late thought then

this method is now public and no longer in context of use, we should give it a more descriptive name

`isHiveSystemSchema` (with inverse semantics)

**Comment by findepi** [comment]
@MiguelWeezardo see build outcome

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java
Comment (findepi) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java:250]: Doesn't belong to `Move isIcebergTable to HiveUtils` commit

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java
Comment (findepi) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java:559]: static import

**Review by findepi — APPROVED** [review]

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java
Comment (findepi) [reply plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java:250]: squash please :)

**Comment by MiguelWeezardo** [comment]
@findepi, @electrum could one of you merge this?

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java
Comment (phd3) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java:378]: nit: `filter(not(HiveUtil::isHiveSystemSchema))`

**Review by phd3 — APPROVED** [review]
Can you please squash the fixup commit and also change the commit message to reflect that we're moving the methods instead of making them public? (Also, I'd avoid mentioning "redirection" in the commit message. We can just say that we're moving out static utility methods.) Will merge the PR once that's resolved.

(There's a minor comment, but would leave it up to you.)

**Comment by findepi** [comment]
squashed myself and merged

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java
Comment (findepi) [reply plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java:378]: @phd3 sorry missed that, didn't refresh the page

**Pull request closed without merging**
